addons/account/tests/test_audit_trail.py

```diff
@@ -43,9 +44,9 @@ def get_trail(self, move):
     def assertTrail(self, trail, expected):
         self.assertEqual(len(trail), len(expected))
         for message, expected in zip(trail, expected[::-1]):
-            self.assertRegex(
+            self.assertIn(
+                expected,
                 html2plaintext(message.account_audit_log_preview),
```

Comment:
The field is no longer HTML therefore I guess this could be simplified ?

Reply:
Done!